lighthouse-cli/chrome-finder.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const execSync = require('child_process').execSync;
+
+module.exports = {
+  darwin() {
+    const suffix = '/Contents/MacOS/Google Chrome Canary';
+
+    const LSREGISTER =
+      '/System/Library/Frameworks/CoreServices.framework' +
+      '/Versions/A/Frameworks/LaunchServices.framework' +
+      '/Versions/A/Support/lsregister';
+
+    const installations = [];
+
+    execSync(
+      `${LSREGISTER} -dump` +
+      ` | grep -i 'google chrome canary.app$'` +
+      ` | awk '{$1=""; print $0}'`
+    ).toString()
+      .split(/\r?\n/)
+      .forEach(inst => {
+        const execPath = path.join(inst.trim(), suffix);
+        if (canAccess(execPath)) {
+          installations.push(execPath);
+        }
+      });
+
+    const priorities = new Map([
+      [/^\/Volumes\//, -1],
+      [/^\/Applications\//, 100],
+      [new RegExp(`^${process.env.HOME}/Applications/`), 50]
+    ]);
+
+    return sort(installations, priorities);
+  },
+
+  linux() {
+    return [process.env.LIGHTHOUSE_CHROMIUM_PATH];
```

Comment:
probably worth copying over [the warning](https://github.com/GoogleChrome/lighthouse/blob/067db41f599503a972cee6e53d9e420fcd106be8/lighthouse-core/scripts/launch-chrome.sh#L46) from `launch-chrome.sh` if this isn't set